Makefile

```diff
@@ -294,11 +294,20 @@ dist/$(PACKAGE_NAME)-$(PACKAGE_VERSION)-docs-html.zip: docs
 dist/$(PACKAGE_NAME)-$(PACKAGE_VERSION)-build-epoch.txt:
 	echo $(SOURCE_DATE_EPOCH) > dist/$(PACKAGE_NAME)-$(PACKAGE_VERSION)-build-epoch.txt
 
-# Build the HTML documentation from the package's source.
+# Build the HTML documentation from RST files under `docs/source`.
```

Comment:
I don't think this change is correct because the documentation is indeed built from the docstrings in the source code.

Reply:
Hmm, I think what I meant here is that: `make docs` does not only build documentation from docstrings but also from other RST files that we write ourselves as well. API references built from docstrings are also generated from skeleton RST files.

Reply:
What do you think about
```suggestion
# Build the HTML documentation from RST files under `docs/source` and Macaron's docstrings.
```

Reply:
I prefer to keep it as it is and revert the change. The previous comment was descriptive enough and this is an unnecessary change.

Reply:
See 5f0691cc04123c86ec35bd362a1c389b0426c1d7.
Although for me "the package's source" sounds like "things under `/src/` and not things under `/docs/source`".